docs/adr/0037-use-nx-for-monorepo.md

```diff
@@ -0,0 +1,48 @@
+# Use nx for monorepo tooling
+
+- Status: proposed
+- Deciders: Ben Bangert, Wil Clouser, Dan Schomburg
+- Date: 2023-03-13
+- Obsoletes: 0019-use-workspace-dependencies
+
+## Context and Problem Statement
+
+Our monorepo using a yarn workspace model has a few limitations we've experienced such as slow builds, using bash scripts for running tasks, having to make each new library or component as its own package with its own dependencies, and an inflexible structure for refactoring our shared libraries.
+
+Monorepo tooling has progressed since the prior ADR on using workspace dependencies with yarn and we'd like to take advantage of these improvements.
+
+## Decision Drivers
+
+- Faster builds with dependency structure analysis and build caching.
+- Easy to migrate into without large changes to our existing setup.
+- Tooling built for monorepo needs.
+- Task running, caching, and dependency handling
+- Easy to create/share/move library/app code for re-use and encapsulation.
```

Comment:
On the topic of consistency, I think quickly scaffolding new apps/services in a standard way that have a consistent structure is also a potential driver. Additionally being able to customize the how an app scaffolds might be a decision driver since we have some customization (thinking l10n here) that is necessary.

Reply:
Yep, definitely, added.